src/common/utxobased/db/makeProcessor.ts

```diff
@@ -106,26 +157,28 @@ async function createOrOpen<T extends BaseType>(
       case BaseType.CountBase:
         return await createCountBase(disklet, config.dbName, config.bucketSize)
       case BaseType.RangeBase:
-        return await createRangeBase(disklet, config.dbName, config.bucketSize, RANGE_KEY, RANGE_ID_KEY)
+        return await createRangeBase(
+          disklet,
+          config.dbName,
+          config.bucketSize,
+          RANGE_KEY,
+          RANGE_ID_KEY
+        )
     }
   } catch (err) {
+    console.log('already exists', config)
```

Comment:
Remove this console.log.
As a general rule, don't use 'console' anywhere other than in the tests.
We will discuss how to properly log things in our meeting on Friday.